html/user/am_set_info.php

```diff
@@ -111,14 +112,27 @@ function success($x) {
 $send_email = BoincDb::escape_string($send_email);
 $show_hosts = BoincDb::escape_string($show_hosts);
 $venue = BoincDb::escape_string($venue);
+$send_changed_email = false;
 if ($email_addr) {
+    $tmpuser = BoincUser::lookup_email_addr($email_addr);
```

Comment:
So, I think what the option you're asking for is.  If email_addr does not equal the same as the user of the auth_token, return an error. if it is the same email_addr for the user with auth, then set email_addr to null for local usage since that is the trigger below in the code.  Does that sound reasonable?